### PR TITLE
Fix to animate_topomap.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/3515

Disables blit when mpl in interactive mode.

### CODE REVIEW

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -2059,7 +2060,13 @@ def _topomap_animation(evoked, ch_type='mag', times=None, frame_rate=None,
         raise ValueError('All times must be inside the evoked time series.')
     frames = [np.abs(evoked.times - time).argmin() for time in times]
 
-    blit = False if plt.get_backend() == 'MacOSX' else blit
+    if blit and plt.isinteractive():
+        logger.info('blit not supported with matplotlib in interactive mode. '
+                    'Disabling blit.')
+        blit = False
+    elif plt.get_backend() == 'MacOSX':
```
Comment (Eric89GXL) [inline mne/viz/topomap.py]: Needs to be
```
elif blit and plt.get_backend() == `MacOSX`
```
But really you can do
```
if blit:
    if ...
    elif ...
```

**Comment by mmagnuski** [comment]
Really happy for the fix, I'll test it soon.

**Comment by agramfort** [comment]
LGTM

**Comment by mmagnuski** [comment]
Unfortunatelly I have the same problems that were raised in original issue:
* "Initializing animation..." is printed every time the animation goes back to the beginning - and the whole animation seems to be recomputed from scratch (the wait time before it plays back is proportional to the number of frames)
* clicking the topo does not stop the progression of time - only turns off redrawing. If I click the topo, and wait a few seconds, and then click again - the animation is a number of frames ahead (proportional to the time being paused)

**Comment by Eric89GXL** [comment]
@mmagnuski did you end up solving this by changing backend?

@jaeilepp do we still want this? Did it fix something at your end, or was it just a prospective fix for @mmagnuski?

**Comment by jaeilepp** [comment]
I would have to set up a Windows environment to take a proper look at this. Closing for now.

**Pull request closed without merging**